drivers/src/persistent.rs

```diff
@@ -36,7 +36,7 @@ pub struct PersistentData {
 }
 impl PersistentData {
     pub fn assert_matches_layout() {
-        const P: *const PersistentData = memory_layout::DCCM_ORG as *const PersistentData;
+        const P: *const PersistentData = memory_layout::MAN1_ORG as *const PersistentData;
```

Comment:
You need to replace other references to DCCM_ORG in this file as well; that's why the tests are failing.

Reply:
Done